lib/phoenix/router/helpers.ex

```diff
@@ -227,6 +227,14 @@ defmodule Phoenix.Router.Helpers do
         endpoint.static_url <> endpoint.static_path(path)
       end
 
+      @doc """
+      Generates an integrity hash to a static asset given its file path.
+      """
+
+      def static_integrity(%Conn{private: %{endpoint: endpoint}}, path) do
```

Comment:
We need to repeat all the three clauses found in `static_path`.

Reply:
ok will fix